kuma/users/fixtures/test_users.json

```diff
@@ -32,6 +32,14 @@
     },
     {
         "fields": {
+            "name": "Trusted writers",
+            "permissions": []
+        },
+        "model": "auth.group",
+        "pk": 3
```

Comment:
``"pk": 3`` is used by the ``Beta Testers`` group earlier in this file, which is probably why the test is failing in TravisCI and in the VM (``py.test kuma/users/tests/test_views.py::UserViewsTest::test_user_edit_beta``)